src/features/input/FieldInput.js

```diff
@@ -0,0 +1,68 @@
+import React, { useState } from "react";
+import { useSelector, useDispatch } from "react-redux";
+import { add, clear, selectInput, addSomeText } from "./inputSlice";
+import "./input.css";
```

Comment:
In the whole project you should use only one type of quotes, either `'` or `"`.
That also can be automated using eslint.